### PR TITLE
Use platform.uname instead of os.uname

### PR DESCRIPTION
**Pull request by Julian-O**
Advantages:

- Works cross platform, not just Unix.
- Is a namedtuple, so can use meaningful  fieldnames.

Also snuck in correction to typo in Readme which doesn't warrant its own review.

[Note: Unit-tests failed, but I don't *think* it had anything to do with this.]

### CODE REVIEW

**Review by misl6 — APPROVED** [review]
CI builds are failing (are now fixed via https://github.com/kivy/python-for-android/pull/2862), but everything is clearly handled as expected on non-sdl2 jobs, so there's no need for another run in this case. (like #2858)

LGTM. Thank you!

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ to install and begin creating APKs and AABs.
 
 (for the develop branch: `pip install git+https://github.com/kivy/python-for-android.git`)
 
-Test that the install works with:
+Test that theinstall works with:
```
Comment (AndreMiras) [inline README.md:42]: oops

Comment (misl6) [reply README.md:42]: Ouch!
For some reason I've read: "Test that install works with:" 🫣

@Julian-O can you take care of it?

Comment (Julian-O) [reply README.md:42]: I just spent some time trying to figure out how I managed to break this comment.

1. Fix a legitimate typo in a comment of no consequence.
2. Accidentally delete a space in another file.
3. A few days later, at commit time, see that delta for the lost space, but don't notice it is the wrong way. Think "Ah, yeah, the typo I fixed. I will include that in this PR."

Sorry. PR with both inconsequential fixes coming very soon.

**Pull request merged**